dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/apps/telemetry/SpanCurrentServlet.java

```diff
@@ -24,10 +24,11 @@
 import jakarta.servlet.annotation.WebServlet;
 
 @SuppressWarnings("serial")
-@WebServlet("/SpanCurrentServlet")
+@WebServlet("/testSpanCurrent")
 public class SpanCurrentServlet extends FATServlet {
 
-    @Test
+    // With HTTP tracing enabled, there will be always a current span.  Comment out this test
+    // @Test
     public void testGetCurrentSpan_Default() {
         Span span = Span.current();
         assertEquals(span, Span.getInvalid()); //Current span has no context as none was created
```

Comment:
Could we replace this with a test that runs a task on an executor without propagating the context and checks the current span there?

Reply:
There is an internal property to turn off the `TelemetryServletFilter`.  Do you want to keep all tests in this server as is and I can add the property to this server.

Reply:
Revised it to under in a task on an executor.